.github/dependabot.yml

```
@@ -3,7 +3,9 @@ updates:
 
 # Maintain dependencies for GitHub Actions
 - package-ecosystem: "github-actions"
-  directory: "/"
+  directories:
+    - "/"
+    - "/.github/actions/*/"
```

I would prefer "proper" indentation, but don't enough for this specific file.
```suggestion
  directories:
  - "/"
  - "/.github/actions/*/"
```